infrastructure/cdn-in-a-box/traffic_ops/aes.conf

```diff
@@ -0,0 +1 @@
+CZhbsfrdNqAC9tfZJ/x4tB/NgoXhbfLdFwauhhNedjE=
```

Comment:
CDN in a Box should generate this. If you want, it can be persisted in a bind mount like the CA directory is:

https://github.com/apache/trafficcontrol/blob/c3878a2a997d48e106ea44bd72e34aa39598e2b9/infrastructure/cdn-in-a-box/docker-compose.yml#L73-L75

But if it does not exist, CiaB should generate it, and no generated key should be tracked in the repo.